charts/kserve-crd/templates/serving.kserve.io_clusterlocalmodels.yaml

```diff
@@ -0,0 +1,86 @@
+---
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
might need to rebase from the master, I think this is already there

Reply:
done